### ISSUE TITLE
Constant too big for 32-bit systems

### ISSUE THREAD

**Issue by jamesjer** [aantron/luv#123]
I attempted to build version 0.5.9 for Fedora (see https://koji.fedoraproject.org/koji/taskinfo?taskID=72934450).  The build failed on our 32-bit architectures (i686 and ARM):

```
File "src/c/luv_c_generated_types.ml", line 771, characters 4-14:
771 |     1143219236
          ^^^^^^^^^^
Error: Integer literal exceeds the range of representable integers of type int
```

The value in question is this one, from src/c/shims.h:
```
#define UV_ESOCKTNOSUPPORT 0x44242424
```

**Comment by aantron** [comment]
Thanks for the report. Could you try with branch `constant-range`, which includes the above commit (https://github.com/aantron/luv/commit/62e0ebc8df8aacf9b601ff64d948158e5cbc7ff4)? If that works, I will release 0.5.10 with it.

This might also be causing #121. cc @RealyUniqueName.

**Comment by jamesjer** [comment]
That branch compiles successfully on our 32-bit architectures.  Thanks for the quick response.

**Comment by aantron** [comment]
Great, thanks for checking! I'll wait perhaps one working day for @RealyUniqueName to try it also. If it fixes the issue he is observing (likely, as it was on i686, looking at the logs), I will make a 0.5.10 release. Otherwise, we might have an additional bug to fix before I'd make the release.

**Comment by RealyUniqueName** [comment]
This fix doesn't work for me

**Comment by aantron** [comment]
The fixed version is now out as Luv 0.5.10.

**Comment by jamesjer** [comment]
Thank you!  I just built it for Fedora a short time ago and the build succeeded.
